party/tribute/ohno-itshimagain.md

```diff
@@ -6,7 +6,41 @@
  * **Combat Role:** Damage + Defense
  * Abilities that sacrifice control and/or risk friendly fire.
 
+### Attributes and Skills
+
+ * **Maximum HP:** 65
+ * **STR:** +5
+ * **CON:** +4
+ * **DEX:** +1
+ * **INT:** -5
+ * **WIS:** -1
+ * **CHA:** +2
+
+### Defenses
+
+ * **REF:** 17
+ * **FORT:** 16
+ * **WILL:** 8
+
+### Abilities
+
+**Hit Them With the Pointy Bits** At-will, Melee, Standard. +3 vs. REF. On hit, deal 1d12 damage.
+
+**But That Looks Like It's Going to Hurt!** At-will, Standard. Gain a +3 to REF and FORT defenses, but suffer a -3 penalty to WILL, until the beginning of your next turn.
+
+**Can't Stop, Won't Stop** Charges: 2, Melee, Standard. +5 vs. REF. On hit, deal 1d12 damage and you *must* repeat this attack on your next turn as a Minor action against a different target if able.
```

Comment:
To be clear: the repeated attack _also_ has this clause. It continues to recur (as a minor action on all but the first) until you miss or run out of valid targets. The targets have to be distinct among all recurring instances, too; you can't just bounce between two enemies.

The intent is to make it really dangerous to use around your allies, because that's kind of a theme here.